flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -382,11 +342,26 @@ private UV deserializeUserValue(byte[] rawValueBytes) {
 			this.rawValueBytes = rawValueBytes;
 			this.deleted = false;
 		}
-		
+
+		public void remove() {
+			deleted = true;
+			rawValueBytes = null;
+
+			try {
+				db.remove(columnFamily, writeOptions, rawKeyBytes);
+			} catch (RocksDBException e) {
+				throw new RuntimeException("Error while removing data from RocksDB.", e);
```

Comment:
I modify the method signature because I find, except `ValueState`, the methods in other states all throw `Exception`. I think it's okay because `MapState` is a common interface which has no idea of the implementation.  The implementation  of these methods, however, should throw some specific exception like `IOException` or `RocksDBException`. 

I think it's reasonable. What do you think?